pytorch_lightning/strategies/hpu_parallel.py

```diff
@@ -47,9 +48,15 @@ def __init__(
         self,
         accelerator: Optional["pl.accelerators.accelerator.Accelerator"] = None,
         parallel_devices: Optional[List[torch.device]] = None,
+        cluster_environment: Optional[ClusterEnvironment] = None,
         checkpoint_io: Optional[CheckpointIO] = None,
         precision_plugin: Optional[PrecisionPlugin] = None,
+        ddp_comm_state: Optional[object] = None,
+        ddp_comm_hook: Optional[callable] = None,  # type: ignore
+        ddp_comm_wrapper: Optional[callable] = None,  # type: ignore
```

Comment:
These callables should be `typing.Callable` and then you shouldn't need that `type: ignore`